### PR TITLE
toptags for users added

### PR DESCRIPTION
**Pull request by Krisselchen**
Added method to get Topten Tags of any user.
Two Endpoints:
api/users/tags    get for current_user
api/users/<email:mail>/tags      get for user belonging to mail

methods are completely identical, suppose it should be "outsourced into helper. But I am tired. :)

### CODE REVIEW

File: backend/api/users.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import sys
+
```
Comment (henderkes) [inline backend/api/users.py]: -- please?

File: backend/api/users.py [hunk 2]
```diff
@@ -139,6 +141,54 @@ def get_user(mail):
...
+    try:
+        pipeline = [{"$unwind": "$authors"},
+                    {"$match": {"authors.email": mail}},
+                    {"$unwind": "$tags"},
+                    {"$group": {"_id": "$tags", "count": {"$sum": 1}}}
+                    ]
+        taglist = sorted(list(g.projects.aggregate(pipeline)), key=lambda k: k['count'],
+                         reverse=True) if g.projects.aggregate(pipeline) else []
+
+        toptags = taglist[0:10] if (len(taglist) > 9) else taglist
```
Comment (henderkes) [inline backend/api/users.py]: if len(taglist) > 9 without braces in python, linter will cry otherwise

File: backend/api/users.py [hunk 3]
```diff
@@ -139,6 +141,54 @@ def get_user(mail):
...
+    try:
+        pipeline = [{"$unwind": "$authors"},
+                    {"$match": {"authors.email": current_user['email']}},
+                    {"$unwind": "$tags"},
+                    {"$group": {"_id": "$tags", "count": {"$sum": 1}}}
+                    ]
+        taglist = sorted(list(g.projects.aggregate(pipeline)), key=lambda k: k['count'],
+                         reverse=True) if g.projects.aggregate(pipeline) else []
+
+        toptags = taglist[0:10] if (len(taglist) > 9) else taglist
```
Comment (henderkes) [inline backend/api/users.py]: same as above

**Review by henderkes — COMMENTED** [review]
rest looks good, does this need testing?

File: backend/api/users.py (hunk 1 above)
Comment (Krisselchen) [reply backend/api/users.py]: Ah sorry

File: backend/api/users.py (hunk 2 above)
Comment (Krisselchen) [reply backend/api/users.py]: linter doesn't isnt running on travis...

Comment (henderkes) [reply backend/api/users.py]: yes but in VS :(

**Comment by Krisselchen** [comment]
I tested in Postman everything worked fine in my test cases.

**Review by Drakulix — DISMISSED** [review]

**Review by henderkes — COMMENTED** [review]
looks great now

**Review by henderkes — APPROVED** [review]

**Pull request merged**